packages/rx_bloc_cli/example/docs/feature_creation.md

```diff
@@ -0,0 +1,208 @@
+## Manual Feature Creation
```

Comment:
@JvnSlv how the developers will access this page ?

Reply:
It can be accessed in the docs directory of the newly generated project. But I guess it should also be linked in the main readme file

Reply:
Fixed here
https://github.com/Prime-Holding/rx_bloc/pull/894

Reply:
The PR is approved